lang/en.yml

```diff
@@ -14,4 +14,4 @@ en:
       other: '{count} files'
     SINGULARNAME: file
   SilverStripe\ElementalBlocks\Form\BlockLinkField:
-    None: None
+    Empty: Empty
```

Comment:
I wonder if elemental has `merge` set for text-collector/transifex
Might be a good idea to keep `None: None` there, or update it with `None: Empty` maybe ping @tractorcow for advice here :)

Reply:
True - @tractorcow?

We could always leave the key the same and change the value.

In this case the original hasn't been pushed to Transifex yet, so it's more of a theoretical question for future :)

Reply:
It's probably ok to just replace it if you never expect to pull translations for the old keys.